templates/tutorials/index.html

```diff
@@ -4,124 +4,156 @@
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1Lpo7Od9VA1aHYmr9UVp02vCGOP5Zxmzx4zZE9YJwlQg/edit{% endblock %}
 
+{% block meta_description %}Learn what you can do with MicroK8s and how, through a series of tutorials on Ubuntu, Windows and macOS using laptops, Raspberry Pis and edge devices.{% endblock %}
+
 {% block content %}
 
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <h1 class="p-heading--2">Tutorials</h1>
+<section class="p-strip u-no-padding--bottom">
+  <div class="p-section--deep">
+    <div class="u-fixed-width">
+      <h1>Tutorials</h1>
+    </div>
   </div>
+</section>
 
-  <div class="row u-equal-height">
-    <div class="col-4 p-card">
-      <div class="p-card__content">
-        <h3 class="p-heading--4"><a href="https://ubuntu.com/tutorials/install-microk8s-on-windows#1-overview">Install MicroK8s on Windows</a></h3>
-        <p>
-          Get a local Kubernetes on Windows with MicroK8s and Multipass.
-        </p>
-      </div>
-      <p>
-        Difficulty: <span class="p-tutorials-meter p-tutorials-meter--1">1 out of 5</span>
+<section class="p-section--deep">
+  <div class="row">
+    <hr class="is-muted">
+    <div class="col-6 col-medium-3">
```

Comment:
we need a "p-strip is-shallow" somewhere here (on all tutorial items)
```suggestion
    <div class="col-6 col-medium-3 p-strip is-shallow">
```

Reply:
I'm not sure I can do that without breaking Vanilla best practices. From the [docs](https://vanillaframework.io/docs/patterns/strip): "A .p-strip container should always be the parent of a .row (from the [Grid pattern](https://vanillaframework.io/docs/patterns/grid/)) and never the other way around."

Reply:
sorry, p-section--shallow is what I meant. No such rigid rules around the p-section.

Reply:
You can put it on a separate div if you prefer, the "suggest" funcitonality in github doesn't give me the whole code, just this line, so I put it like that. The important thing is to apply a shallow spacing at the bottom (only, not the top) of each tutorial item.